.github/CODEOWNERS

```diff
@@ -3,7 +3,7 @@
 
 # Chris Wilcox is the default owner for anything not explicitly
```

Comment:
Can we change the comment as well for consistency/remove it?

Reply:
I updated the comment.  Good catch.